From 4bb5f30e0f8e81ace3fbedcbe93347ce9db2ea22 Mon Sep 17 00:00:00 2001
From: Clemens Lang <cllang@redhat.com>
Date: Mon, 5 Oct 2026 10:00:52 +0200
Subject: [PATCH] Makefile: Use gsed, ggrep, mandoc on macOS, too

macOS ships with BSD sed, BSD grep and a version of man that does not
support -L. Give it the same treatment as FreeBSD to avoid warnings
during the build and potentially incorrect output.

Signed-off-by: Clemens Lang <cllang@redhat.com>
Upstream-Status: Submitted [https://github.com/podman-container-tools/podman/pull/29900]
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index cc1c6af239..14f111b93e 100644
--- ./Makefile
+++ ./Makefile
@@ -77,7 +77,7 @@ PYTHON ?= $(shell command -v python3 python|head -n1)
 PKG_MANAGER ?= $(shell command -v dnf yum|head -n1)
 # ~/.local/bin is not in PATH on all systems
 PRE_COMMIT = $(shell command -v bin/venv/bin/pre-commit ~/.local/bin/pre-commit pre-commit | head -n1)
-ifeq ($(NATIVE_GOOS),freebsd)
+ifneq (,$(filter $(NATIVE_GOOS),freebsd darwin))
 SED=gsed
 GREP=ggrep
 MAN_L=	mandoc
-- 
2.56.0

